modules/gridBidAdapter.js

```diff
@@ -224,75 +129,403 @@ function _getBidFromResponse(respItem) {
+      pageKeywords = utils.transformBidderParamKeywords(keywords);
+    }
+    if (realTimeData && realTimeData.jwTargeting) {
+      if (!jwpseg && realTimeData.jwTargeting.segments) {
+        jwpseg = realTimeData.segments;
+      }
+      if (!content && realTimeData.content) {
+        content = realTimeData.content;
+      }
+    }
+    let impObj = {
+      id: bidId,
+      tagid: uid.toString(),
+      ext: {
+        divid: adUnitCode
+      }
+    };
+
+    if (!mediaTypes || mediaTypes[BANNER]) {
+      const banner = createBannerRequest(bid, mediaTypes ? mediaTypes[BANNER] : {});
+      if (banner) {
+        impObj.banner = banner;
+      }
+    }
+    if (mediaTypes && mediaTypes[VIDEO]) {
+      const video = createVideoRequest(bid, mediaTypes[VIDEO]);
+      if (video) {
+        impObj.video = video;
+      }
+    }
+
+    if (impObj.banner || impObj.video) {
+      imp.push(impObj);
+    }
+  });
+
+  const source = {
+    tid: auctionId,
+    ext: {
+      wrapper: 'Prebid_js',
+      wrapper_version: '$prebid.version$'
+    }
+  };
+
+  if (schain) {
+    source.ext.schain = schain;
+  }
+
+  const tmax = config.getConfig('bidderTimeout') || timeout;
+
+  let request = {
+    id: bidderRequestId,
+    site: {
+      page: referer
+    },
+    tmax,
+    source,
+    imp
+  };
+
+  if (content) {
+    request.site.content = content;
+  }
+
+  if (jwpseg && jwpseg.length) {
+    user = {
+      data: [{
+        name: 'iow_labs_pub_data',
+        segment: jwpseg.map((seg) => {
+          return {name: 'jwpseg', value: seg};
+        })
+      }]
+    };
+  }
+
+  if (gdprConsent && gdprConsent.consentString) {
+    userExt = {consent: gdprConsent.consentString};
+  }
+
+  if (userId) {
+    userExt = userExt || {};
+    if (userId.tdid) {
+      userExt.unifiedid = userId.tdid;
+    }
+    if (userId.id5id) {
+      userExt.id5id = userId.id5id;
+    }
+    if (userId.digitrustid && userId.digitrustid.data && userId.digitrustid.data.id) {
+      userExt.digitrustid = userId.digitrustid.data.id;
+    }
+    if (userId.lipb && userId.lipb.lipbid) {
+      userExt.liveintentid = userId.lipb.lipbid;
+    }
+  }
+
+  if (userExt && Object.keys(userExt).length) {
+    user = user || {};
+    user.ext = userExt;
+  }
+
+  if (user) {
+    request.user = user;
+  }
+
+  const configKeywords = utils.transformBidderParamKeywords({
+    'user': utils.deepAccess(config.getConfig('fpd.user'), 'keywords') || null,
+    'context': utils.deepAccess(config.getConfig('fpd.context'), 'keywords') || null
+  });
+
+  if (configKeywords.length) {
+    pageKeywords = (pageKeywords || []).concat(configKeywords);
+  }
+
+  if (pageKeywords && pageKeywords.length > 0) {
+    pageKeywords.forEach(deleteValues);
+  }
+
+  if (pageKeywords) {
+    request.ext = {
+      keywords: pageKeywords
+    };
+  }
+
+  if (gdprConsent && gdprConsent.gdprApplies) {
+    request.regs = {
+      ext: {
+        gdpr: gdprConsent.gdprApplies ? 1 : 0
```

Comment:
Your tests are expecting boolean, not number.  
See output of circleci, it is failing and I think it is because of this line.